### ISSUE TITLE
RadSSH on Windows platform

### ISSUE THREAD

**Issue by chicjo01** [radssh/radssh#27]
During the installation of the colorama branch.

*  I had to upgrade to the lastest pip version:
pip install pip --upgrade (Needed to have administrator rights)

* Readline module is not installed by default, need to install pyreadline
pip install pyreadline

* Attempting to run RadSSH, I encountered a TypeError, needed to install the latest colorama version
GitHub Download of colorama
python setup.py install

**Comment by radssh** [comment]
Thanks for the report. There are a number of issues related to getting RadSSH properly functional on the Windows platform (which is why that in-progress code is isolated on its own branch, for now). IMO, it won't be releasable until the colorama fix is included in a public release, so I can slap a minimum version requirement.

The other issues you include will also need to be addressed, and that list will probably grow as more testing of a native windows platform release gets done over time. I'd like to keep this issue open as a collection bucket for Windows/colorama related issues.

If you don't find that branch stable enough, you might try running RadSSH under the Win10/Bash for Windows environment. I know of some users having good success with that.

**Comment by radssh** [comment]
Still waiting on a formal release of colorama (with crucial bug fix) before I can seriously consider pushing this to master for inclusion.

**Comment by radssh** [comment]
Formal support for windows platform included in 1.1.2. Any discovered issues with that (or later) release, please open a new Issue.
